packages/lesswrong/lib/collections/ckEditorUserSessions/schema.ts

```diff
@@ -0,0 +1,25 @@
+import { accessFilterSingle, resolverOnlyField, schemaDefaultValue } from "../../utils/schemaUtils";
+import {userOwns} from "../../vulcan-users/permissions";
+
+const schema: SchemaType<DbCkEditorUserSession> = {
+  documentId: {
+    type: String,
+    nullable: false,
+    canRead: ['members'],
+    canCreate: ['members'],
+  },
+  userId: {
+    type: String,
+    nullable: false,
+    canRead: ['members'], 
+    canCreate: ['members'],
+  },
+  endedAt: {
+    type: Date,
+    optional: true,
+    canRead: ['members'],
+    canCreate: ['members'],
```

Comment:
We can probably omit `canCreate`, since we aren't expecting users to be creating these records directly anyways (so giving them permission to create individual fields is just a footgun, in case we ever do give users permission to create the records in the future).

Reply:
Also, `canRead` should probably be restricted to `admin`, if we're even allowing users to read these records through default resolvers (which it's not obvious we should; it seems like we're mostly doing all the querying on this table directly from the backend).